kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkBatchProcessBuilder.scala

```diff
@@ -44,16 +45,14 @@ class SparkBatchProcessBuilder(
       buffer += cla
     }
 
-    // tag for YARN
-    val batchJobTag = batchConf.get(TAG_KEY).map(_ + ",").getOrElse("") + batchId
-    var allConf = batchConf ++ Map(TAG_KEY -> batchJobTag) ++ sparkAppNameConf()
+    // session conf + batch conf
+    // user set batch conf should override session conf
+    batchConf.asJava.entrySet().forEach(entry => conf.set(entry.getKey, entry.getValue))
```

Comment:
we should not use the sessionConf(KyuubiConf), it might contain the spark parameters set for interactive session.

Reply:
and it will also introduce many kyuubi parameters for the batch job

Reply:
I see, so it means we have to modify func `KyuubiApplicationManager#tagApplication` for supporting Map type. Or we can add new func to support Map params.

Reply:
how about:
```
    val tagConf = new KyuubiConf(false)
    KyuubiApplicationManager.tagApplication(batchId, "spark", clusterManager(), tagConf)
    val allConf = batchConf ++ tagConf.getAll
    allConf.foreach { case (k, v) =>
      buffer += CONF
      buffer += s"$k=$v"
    }

```

Reply:
This also seems to introduce Kyuubi configurations. 
I have made a new change, please check whether this is reasonable.

Reply:
seems the settings of `new KyuubiConf(false)` is empty